### DISCUSSION TITLE
引入自己写的组件（上传至npm上的）后，拖入画布不显示

### DISCUSSION THREAD

**Discussion by kalvinhu** [opentiny/tiny-engine#587] [Q&A]
组件地址：https://unpkg.com/hrj-bar-chart@1.0.6/kalvin/kvn-barchart.esm.js

以下文件是在 materials/components 下新建的物料json文件
[KvnBarchart.json](https://github.com/user-attachments/files/15909240/KvnBarchart.json)

**Comment by wenmine** [comment]
@kalvinhu 物料的配置看起来没有问题，组件的打包和导出有问题，![image](https://github.com/opentiny/tiny-engine/assets/14882982/83033574-7df5-4b17-aabb-9a81aa04dc46)

**Reply by wenmine** [reply]
KvnBarchart并没有被导出

**Reply by kalvinhu** [reply]
嗯嗯，问题已经解决，谢啦
